### PR TITLE
Added message for low funds

### PR DESCRIPTION
**Pull request by joandrsn**
If you have under a balance under 5000, you should get a message, notifying you that you might soon be banned.

### CODE REVIEW

File: stregsystem/templates/stregsystem/index_sale.html [hunk 1]
```diff
@@ -15,6 +15,12 @@
 {{product.name}}{% endfor %}
 for tilsammen {{cost|money}} kr.
 </b>
+{% if showmemberbalance %}
+<br>
+<b>
+    Tilbagestående beløb: {{newmemberbalance}}
```
Comment (VirtualSatai) [inline stregsystem/templates/stregsystem/index_sale.html]: Consider adding a hint that they should insert more money soon.

**Comment by jenrik** [comment]
I don't think we should leave peoples balance visible, I think either there should be timeout after which the message disappear, or the message should only state that the account balance is below 5000, or both

**Comment by joandrsn** [comment]
Agree. I would like to wait for #182 to be merged.

**Comment by jenrik** [comment]
@joandrsn #182 has been merged, should we look at getting this merged too?

**Comment by joandrsn** [comment]
I have updated the code to remove the remaning amount, using the same method as in #182 
Maybe @hrjakobsen or @cogitantium can review this PR?

File: stregsystem/templates/stregsystem/index_sale.html [hunk 2]
```diff
@@ -15,6 +15,12 @@
 {{product.name}}{% endfor %}
 for tilsammen {{cost|money}} kr.
 </b>
+{% if showmemberbalance %}
+<br>
+<b>
+    Tilbagestående beløb: <span class="remainingbalance">{{newmemberbalance}}</span>
```
Comment (cogitantium) [inline stregsystem/templates/stregsystem/index_sale.html:21]:
```suggestion
    Balance: <span class="remainingbalance">{{newmemberbalance}}</span>
```

This is just my preference

**Review by cogitantium — CHANGES_REQUESTED** [review]

**Review by cogitantium — APPROVED** [review]
LGTM :ok_hand:

**Review by jenrik — APPROVED** [review]

**Comment by JakobTopholt** [comment]
LGTM

**Comment by falkecarlsen** [comment]
@joandrsn made feature catch up and added blinking, note that only Chrome-based browsers load `gdpr.js` for some reason.

**Pull request merged**